wasm/notebook/snippets/python-js-css/javascript.txt

```diff
@@ -1,45 +1,25 @@
-// Javascript code go here
+// Javascript code goes here
 
-// Javascript code go here
+injectPython({
+    // injectPython functions take the positional arguments as
```

Comment:
thanks for adding a decent way to handle arguments. should we add a note about this in the README for the notebook?